allauth/templates/mfa/email/totp_activated_subject.txt

```diff
@@ -0,0 +1,4 @@
+{% load i18n %}
+{% autoescape off %}
+{% blocktrans %}Totp activated.{% endblocktrans %}
```

Comment:
To bring this in line with the terminology used in the messages:

```suggestion
{% blocktrans %}Authenticator App Activated{% endblocktrans %}
```

Reply:
replaced